src/Files.App/Dialogs/FileTooLargeDialog.xaml

```diff
@@ -0,0 +1,36 @@
+﻿<!--  Copyright (c) 2023 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ContentDialog
+	x:Class="Files.App.Dialogs.FileTooLargeDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	Title="{helpers:ResourceString Name=StatusCopyFailed}"
```

Comment:
Is there something a little more user friendly or is "Copy failed" good enough?

Reply:
"CopyFailedMissingDiskSpace" ?
"CopyFailureFileToLarge" ?

Reply:
> "CopyFailedMissingDiskSpace" ? "CopyFailureFileToLarge" ?

I prefer the second one

Reply:
Something like this `Copy has failed because some files are too large`?

Reply:
I propose "Copy didn't succeed" or to be more precise "Copy didn't succeed. File is too large."

Reply:
> "Copy didn't succeed. File(s) is(are) too large."

This one sound good. What do you think @QuaintMako @yaira2?

Reply:
Good for me!

Reply:
"The following items are too large to be copied to this drive"

Reply:
"Cancel", "Skip"

Reply:
> "The following items are too large to be copied to this drive"

Isn't this redundant? It's almost the same as the subtitle

Reply:
> "Cancel" "Skip"

Skip isn't necessary. Valid items will be copied anyway

Reply:
> Isn't this redundant? It's almost the same as the subtitle

Can we remove the subtitle?